### DISCUSSION TITLE
<title>我无法证明我是学生

### DISCUSSION THREAD

**Discussion by Qingquan-serene** [community/community#40995] [GitHub Education]
### Select Topic Area

Show & Tell

### Body

My school did not give me a student ID card or proof of enrolment and I cannot verify that I am a student
The tips are as follows：
Your application cannot be reviewed until you fix the following:
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.,If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application). ,You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.

**Comment by samus-aran** [comment]
Hi @cqy20091031mc 👋 

We're happy you're here! Thanks for sharing your rejection reasons. Try and follow each one as best as you can and amend your application to resolve them. It says that you don't have your school name in the evidence - that's important.  

Although you may not have a Student ID you should be able to apply with an alternative evidence. Here are some tips if you are using alternative evidence:

1. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Dated Student ID being at the top. If you don't have a Student ID do try and acquire evidence that is positioned higher rather than lower in that option list.

2. Ensure your image (photo or upload) is eligible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Hope that helps. 👍
